prog/github/repository_nexus.rb

```diff
@@ -25,6 +25,13 @@ def client
   def check_queued_jobs
     queued_runs = client.repository_workflow_runs(github_repository.name, {status: "queued"})[:workflow_runs]
     Clog.emit("polled queued runs") { {polled_queued_runs: {repository_name: github_repository.name, count: queued_runs.count}} }
+
+    remaining_quota = client.rate_limit.remaining / client.rate_limit.limit.to_f
+    if remaining_quota < 0.1
```

Comment:
do we have an estimate on how many API calls we make at the moment only to register runners? It would be nice to know the max usage only to operate this product and then decide on these percentages.

Reply:
In an ideal scenario, we send 3 API calls for a runner, but I don't have a total estimate. Meanwhile, each GitHub installation has a varying rate limit, determined by the repository count, account plan, and user count, which complicates the process of determining accurate percentages. However, the `remaining_quota < 0.5` check significantly reduces the polling interval. I don't expect to reach the `0.1` limit.